demo/src/app/pages/demo/components/custom-column.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, ViewChild, ElementRef } from '@angular/core';
```

Comment:
Generally, it's a great component idea! The only thing is that probably it's a bit too complex for a basic showcase? So it may be a bit difficult to get an idea of custom editor through this example. Probably we can simplify this into something less complex?